integration/pid.tmpl

```diff
@@ -0,0 +1 @@
+{{ .ProcessID }}
```

Comment:
just an idea, if you don't like it it's fine: instead of creating files for each template, which is not going to be manageable if we have lots of tests, we could just create a tmp file with the test-specific template within the test the needs it.

Reply:
Good point, this was on my mind as well. I updated here https://github.com/aquasecurity/tracee/pull/606/commits/b98c571f2f2af083cec43dc7a301ac556d40077d